From: Roland Stigge <stigge@antcom.de>
Date: Sun, 10 Jul 2011 12:19:43 +0200
Subject: doc remove nonfree options

Remove documentation of nonfree options

Non-free options are not supported by the Debian packaging of
pstoedit.

This patch removes the documentation of some nonfree options that confused
users (plot-svg vs. svg). NOTE: When using this patch, make sure to regenerate
the man page and HTML documentation from the tex file also, e.g. with
latex2man.

Bug-Debian: http://bugs.debian.org/554518
---
 doc/pstoedit.tex | 140 -------------------------------------------------------
 1 file changed, 140 deletions(-)

diff --git a/doc/pstoedit.tex b/doc/pstoedit.tex
index b2856e0..2752fa3 100755
--- a/doc/pstoedit.tex
+++ b/doc/pstoedit.tex
@@ -595,54 +595,6 @@ cubic ???
 trace ???
 
 
-\end{description}
-%%// end of options
-\subsubsection{svg - Scalable Vector Graphics}
-\begin{description}
-\item[\oOpt{-localdtd}]
-use local DTD
-
-
-\item[\oOpt{-standalone}]
-create stand-alone type svg
-
-
-\item[\oOpt{-withdtd}]
-write DTD
-
-
-\item[\oOpt{-withgrouping}]
-write also ordinary save/restores as SVG group
-
-
-\item[\oOpt{-nogroupedpath}]
-do not write a group around paths
-
-
-\item[\oOpt{-noviewbox}]
-do not write a view box
-
-
-\item[\oOpt{-texmode}]
-\TeX\, mode
-
-
-\item[\oOpt{-imagetofile}]
-write raster images to separate files instead of embedding them
-
-
-\item[\oOpt{-notextrendering}]
-do not write textrendering attribute
-
-
-\item[\oOptArg{-border}{~number}]
-additional border to draw around bare bounding box (in percent of width and height)
-
-
-\item[\oOptArg{-title}{~string}]
-text to use as title for the generated document
-
-
 \end{description}
 %%// end of options
 \subsubsection{xaml - eXtensible Application Markup Language}
@@ -691,98 +643,6 @@ additional border to draw around bare bounding box (in percent of width and heig
 text to use as title for the generated document
 
 
-\end{description}
-%%// end of options
-\subsubsection{cgmb1 - CGM Binary format (V1)}
-No driver specific options
-%%// end of options
-\subsubsection{cgmb - CGM Binary format (V3)}
-No driver specific options
-%%// end of options
-\subsubsection{cgmt - CGM Textual format}
-No driver specific options
-%%// end of options
-\subsubsection{mif - (Frame)Maker Intermediate Format}
-\begin{description}
-\item[\oOpt{-nopage}]
-do not add a separate Page entry
-
-
-\end{description}
-%%// end of options
-\subsubsection{rtf - Rich Text Format}
-No driver specific options
-%%// end of options
-\subsubsection{wemf - Wogl's version of EMF}
-\begin{description}
-\item[\oOpt{-df}]
-write info about font processing
-
-
-\item[\oOpt{-dumpfontmap}]
-write info about font mapping
-
-
-\item[\oOpt{-size:psbbox}]
-use the bounding box as calculated by the PostScript frontend as size
-
-
-\item[\oOpt{-size:fullpage}]
-set the size to that of the full page
-
-
-\item[\oOpt{-size:automatic}]
-let MS Windows calculate the bounding box (default)
-
-
-\item[\oOpt{-keepimages}]
-debug option - keep the embedded bitmaps as external files
-
-
-\item[\oOpt{-useoldpolydraw}]
-do not use MS Windows' PolyDraw but an emulation of it - sometimes needed for certain programs reading the EMF files
-
-
-\item[\oOpt{-OO}]
-generate OpenOffice compatible EMF file
-
-
-\end{description}
-%%// end of options
-\subsubsection{wemfc - Wogl's version of EMF with experimental clip support}
-\begin{description}
-\item[\oOpt{-df}]
-write info about font processing
-
-
-\item[\oOpt{-dumpfontmap}]
-write info about font mapping
-
-
-\item[\oOpt{-size:psbbox}]
-use the bounding box as calculated by the PostScript frontend as size
-
-
-\item[\oOpt{-size:fullpage}]
-set the size to that of the full page
-
-
-\item[\oOpt{-size:automatic}]
-let MS Windows calculate the bounding box (default)
-
-
-\item[\oOpt{-keepimages}]
-debug option - keep the embedded bitmaps as external files
-
-
-\item[\oOpt{-useoldpolydraw}]
-do not use MS Windows' PolyDraw but an emulation of it - sometimes needed for certain programs reading the EMF files
-
-
-\item[\oOpt{-OO}]
-generate OpenOffice compatible EMF file
-
-
 \end{description}
 %%// end of options
 \subsubsection{wemfnss - Wogl's version of EMF - no subpaths}
